src/test_lib_json/main.cpp

```diff
@@ -2607,6 +2620,8 @@ int main(int argc, const char* argv[]) {
 
   JSONTEST_REGISTER_FIXTURE(runner, RValueTest, moveConstruction);
 
```

Comment:
Nit: remove extra newline.

Reply:
Are you referring to the newline before or after line 2621? Or are you referring to line 2623?

Reply:
I think this is fine as-is actually, the fixture spacing is just excessive.